extensions/cli/geoserver/src/main/java/mil/nga/giat/geowave/cli/geoserver/GeoServerListCoverageStoresCommand.java

```diff
@@ -10,43 +10,50 @@
  ******************************************************************************/
 package mil.nga.giat.geowave.cli.geoserver;
 
+import java.io.File;
+
 import javax.ws.rs.core.Response;
 import javax.ws.rs.core.Response.Status;
 
 import mil.nga.giat.geowave.core.cli.annotations.GeowaveOperation;
 import mil.nga.giat.geowave.core.cli.api.Command;
 import mil.nga.giat.geowave.core.cli.api.DefaultOperation;
 import mil.nga.giat.geowave.core.cli.api.OperationParams;
-
+import mil.nga.giat.geowave.core.cli.operations.config.options.ConfigOptions;
 import net.sf.json.JSONArray;
 import net.sf.json.JSONObject;
 
 import com.beust.jcommander.Parameter;
 import com.beust.jcommander.Parameters;
 
-@GeowaveOperation(name = "listcs", parentOperation = GeoServerSection.class)
+@GeowaveOperation(name = "listcs", parentOperation = GeoServerSection.class, restEnabled = GeowaveOperation.RestEnabledType.POST)
 @Parameters(commandDescription = "List GeoServer coverage stores")
 public class GeoServerListCoverageStoresCommand extends
-		DefaultOperation implements
+		DefaultOperation<String> implements
 		Command
 {
 	private GeoServerRestClient geoserverClient = null;
 
 	@Parameter(names = {
 		"-ws",
 		"--workspace"
-	}, description = "<workspace name>")
+	}, required = false, description = "<workspace name>")
```

Comment:
its fine to be explicit, but it does default to `required = false` so 99% of the time we just leave it as implicit